src/main/java/uk/gov/hmcts/reform/enforcement/testingsupport/endpoint/TestingSupportController.java

```diff
@@ -21,12 +21,12 @@
 
 @Slf4j
 @RestController
-@RequestMapping("/notify")
-public class NotifyController {
+@RequestMapping("/testing-support")
+public class TestingSupportController {
```

Comment:
Can we make this optional based on a property please and disable it by default. Idea is these end points shouldn't be enabled in prod.